packages/store-indexer/src/postgres/apiRoutes.ts

```diff
@@ -0,0 +1,42 @@
+import { Sql } from "postgres";
+import { Middleware } from "koa";
+import Router from "@koa/router";
+import compose from "koa-compose";
+import { input } from "@latticexyz/store-sync/indexer-client";
+import { storeTables } from "@latticexyz/store-sync";
+import { queryLogs } from "./queryLogs";
+import { recordToLog } from "./recordToLog";
+import { debug } from "../debug";
+import { createBenchmark } from "@latticexyz/common";
+import { compress } from "../compress";
+
+export function apiRoutes(database: Sql): Middleware {
+  const router = new Router();
+
+  router.get("/api/logs", compress(), async (ctx) => {
+    const benchmark = createBenchmark("postgres:logs");
+
+    try {
+      const opts = input.parse(typeof ctx.query.input === "string" ? JSON.parse(ctx.query.input) : {});
```

Comment:
if this parse fails, it should prob be a 4xx error (client error) rather than a 5xx error (server error)